### PR TITLE
Fixes #32602 -- Clarify wording re: parallel testing and test case vs. test case class

### PR DESCRIPTION
**Pull request by MandadiMahendharReddy**
Ticket link: https://code.djangoproject.com/ticket/32602

### CODE REVIEW

File: django/test/runner.py [hunk 1]
```diff
@@ -1179,7 +1180,7 @@ def reorder_tests(tests, classes, reverse=False, shuffler=None):
         yield from reorder_test_bin(tests, shuffler=shuffler, reverse=reverse)
 
 
-def partition_suite_by_case(suite):
+def partition_suite_by_test_class(suite):
```
Comment (felixxm) [inline django/test/runner.py:1183]: Please revert this change. PR should be limited to docs changes.

File: docs/ref/django-admin.txt [hunk 2]
```diff
@@ -1484,8 +1484,7 @@ override this by passing the desired number of processes, e.g.
 variable.
 
 Django distributes test cases — :class:`unittest.TestCase` subclasses — to
-subprocesses. If there are fewer test cases than configured processes, Django
-will reduce the number of processes accordingly.
```
Comment (felixxm) [inline docs/ref/django-admin.txt:1488]: Why you removed: _"Django will reduce the number of processes accordingly"_? This ticket is about clarifying wording `test`/`test case`/`test case class`, etc.

Comment (MandadiMahendharReddy) [reply docs/ref/django-admin.txt:1488]: Hi @felixxm,as per this
image I replaced that line with "If there are fewer test case classes than configured processes."

File: django/test/runner.py (hunk 1 above)
Comment (MandadiMahendharReddy) [reply django/test/runner.py]: Reverted

File: docs/ref/django-admin.txt (hunk 2 above)
Comment (felixxm) [reply docs/ref/django-admin.txt:1488]: Again, this ticket about clarifying wording `test`/`test case`/`test case class`, etc. The sentence that you removed is correct and should be kept.

Comment (felixxm) [reply docs/ref/django-admin.txt:1488]: Also, this is only an example from the ticket description, you should audit all docs and check used wording ( `test`/`test case`/`test case class`).

**Comment by felixxm** [comment]
Closing due to inactivity.
